src/features/ProtectTransaction/__tests__/utils.spec.ts

```diff
@@ -1,67 +1,92 @@
-import { ProtectTxUtils, NansenReportType } from '..';
+import {
+  NansenReportType,
+  getProtectTransactionFee,
+  getNansenReportType,
+  getLastTx,
+  getBalance
+} from '..';
 import { GetTxResponse, GetTokenTxResponse, GetBalanceResponse } from '@services';
 
+describe('getProtectTransactionFee', () => {
+  const rate = 250.5;
+  const formValues = {
+    amount: '0.01',
+    gasEstimates: { safeLow: 65 },
+    gasLimitField: '21000',
+    gasPriceField: '20',
+    gasPriceSlider: '72',
+    advancedTransaction: false
+  };
+  it('should return correctly calculated values', () => {
```

Comment:
active voice: `it('returns ...')`
specific: correctly is a bit vague. Describe the expected operation. e.g. `it('extracts the fee from the amount') ?